src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -214,6 +214,7 @@ protected function configureOptions(OptionsResolver $resolver): void
             ->setDefault('product_attributes', [])
             ->setAllowedTypes('product_attributes', 'array')
             ->setNormalizer('product_attributes', function (Options $options, array $productAttributes): array {
+                $productAttributes = $this->filterTranslationsFromAttributes($productAttributes);
```

Comment:
Why is it a part of this PR? This change seems to not be related to translation at all

Reply:
I was not completely sure what happened, but the tests failed without it. I think the translations were somehow interpreted as a product attribute, but since they lacked the options of a attribute, the test failed.

Reply:
Can you post the exact error that happened here? I'm worrying that we will hide some issues with this change.

Reply:
As next week we will have a feature freeze, can you provide this rather ASAP?

Reply:
```
TypeError: Argument 1 passed to Sylius\Bundle\CoreBundle\Fixture\Factory\ProductExampleFactory::configureProductAttributeValue() must be of the type string, int given, called in ProductExampleFactory.php on line 382
```